widget/ui/src/components/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,24 @@
+import type { PropTypes } from './NotFound.types';
+
+import React from 'react';
+
+import { SearchIcon } from '../../icons';
+import { Divider } from '../Divider';
+import { Typography } from '../Typography';
+
+import { Container } from './NotFound.styles';
+
+export function NotFound({ title, subTitle }: PropTypes) {
+  return (
+    <Container>
+      <SearchIcon color="primary" size={26} />
```

Comment:
The search Icon in Figma has a 45px wrapper division. Also, its color is different

Reply:
Search icon with padding around it's 45 px without space around it's 26 px